code401challenges/src/test/java/tree/TreeTest.java

```diff
@@ -111,4 +111,18 @@ public void testInOrder() {
         assertEquals(post, actual);
     }
 
+    @Test
+    public void testMax(){
+        BinarySearchTree bst = new BinarySearchTree();
```

Comment:
This was a good idea, but this method should work on all trees, not just a binarySearchTree (because that's too easy!)